features/user_can_asign_categories_to_articles.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Better orginized websitet with categories
+  As a subscriber
+  In order to be able to find content of interest
+  I would like to be able to filter articles by category
+
+Background:
+  Given the following article exists
+    | title               | content                       |
+    | A Whole New World   | A new fantastic point of view |
```

Comment:
Why do you have this background step here?

Reply:
Wasn't sure how we would make it write the test in the beginning. Will remove it!